govlearn-api/src/main/java/com/unimuenster/govlearnapi/tags/controller/TagsController.java

```diff
@@ -31,7 +33,9 @@ public class TagsController {
 
```

Comment:
Den Tags Controller evtl. nochmal aufteilen, weil man in Swagger UI keine Übersicht mehr hat. Aber im gleichen Ordner  ist gut (vlt. nach User, Courses und Tags). Außerdem gibt es jetzt einen /courses/tags/{id} und /tags/courses/{id} Endpunkt.

Reply:
habe das jetzt so, dass der erste Fahrt angibt was man zurückbekommt. Also bei courses/tags/{id} werden Kurse zurückgegeben und bei tags/ eben tags zu der Kurs ID. Meinst du Endpunkte coursetag/  und usertag/  sind besser für die Übersichtlichkeit?

Reply:
Würde das erstmal in einzelne Controller zerteilen, damit im Swagger UI mehrere Überschriften entstehen. Würde das jetzt nicht usertag oder so nennen. Versuch die Bedeutung einfach in der Beschreibung von Swagger UI alles möglichst eindeutig zu machen.

Reply:
Habe jetzt drei Controller draus gemacht. Den Tag-Controller nur mit simplen Tag dingen und dann UserTag und CourseTag ohne die Endpunkte zu ändern